### ISSUE TITLE
Invalid @keyframe rules generated in Vue scoped styles

### ISSUE THREAD

**Issue by ahku** [windicss/vite-plugin-windicss#158]
**Describe the bug**
It seems like the [data-v-xxx] attribute is added to the keyframe keys, producing invalid rules. Seem output below.

![CleanShot 2021-04-30 at 09 13 53](https://user-images.githubusercontent.com/1736890/116661731-35a29200-a995-11eb-95b2-fe5c98cd14bb.png)

**Minimal Reproductions**

This CSS
```html
<style scoped>
.test {
  animation: my-animation 2s infinite linear;
}
@-webkit-keyframes my-animation {
  100% {
    -webkit-transform: rotate(360deg);
  }
}
</style>
```

Output:
```css
.test[data-v-0de3663a] {
  animation: my-animation-0de3663a 2s infinite linear;
}
@-webkit-keyframes my-animation-0de3663a {
  100%[data-v-0de3663a] {
      -webkit-transform: rotate(360deg);
  }
}
```

**Versions**
- vite-plugin-windicss: 0.16.0-beta.4
- vite: 2.2.3
- framework(vue/react/svelte/etc): Vue

**Additonal Context**

**Comment by antfu** [comment]
Em, not sure, is this working fine without windi? By looking at it, I don't think that Windi did any transformations on this 🤔

**Comment by harlan-zw** [comment]
Similar to this https://github.com/windicss/windicss/issues/212 ?

**Comment by antfu** [comment]
Probably fixed with #169, can you check it out?
